examples/crypto/crypto-btc-usd-swing-trade.ipynb

```diff
@@ -0,0 +1,431 @@
+    "    TimeInForce,\n",
+    "    QueryOrderStatus\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Please change the following to your own PAPER api key and secret\n",
+    "# You can get them from https://alpaca.markets/\n",
+    "\n",
+    "API_KEY = \"<YOUR PAPER API KEY>\"\n",
+    "API_SECRET = \"<YOUR PAPER API SECRET>\"\n",
+    "\n",
+    "#### We use paper environment for this example ####\n",
+    "PAPER = True # Please do not modify this. This example is for paper trading only.\n",
+    "####\n",
+    "\n",
+    "# Below are the variables for development this documents\n",
+    "# Please do not change these variables\n",
+    "trade_api_url = None\n",
+    "trade_api_wss = None\n",
+    "data_api_url = None\n",
+    "stream_data_wss = None\n",
+    "\n",
+    "# setup trading clients\n",
+    "trade_client = TradingClient(api_key=API_KEY, secret_key=API_SECRET, paper=PAPER, url_override=trade_api_url)\n",
+    "# setup crypto historical data client\n",
+    "crypto_historical_data_client = CryptoHistoricalDataClient(api_key=API_KEY, secret_key=API_SECRET)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Step 2: Executing Market Data Analysis"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# we will place orders which Alapca trading platform supports\n",
+    "# - order types for crypto: market, limit, stop_limit\n",
+    "# - time_in_force for crypto: gtc, ioc.\n",
+    "# please refer to the following documents for more details\n",
+    "# ref. https://docs.alpaca.markets/docs/orders-at-alpaca\n",
+    "# ref. https://docs.alpaca.markets/reference/postorder\n",
+    "\n",
+    "# we will place orders for symbol: BTC/USD in this example\n",
+    "symbol = \"BTC/USD\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set the timezone\n",
+    "timezone = ZoneInfo(\"America/New_York\")\n",
+    "\n",
+    "# Get current date in US/Eastern timezone\n",
+    "today = datetime.now(timezone).date()\n",
+    "\n",
+    "# Define the start and end dates for the historical period\n",
+    "start_date = today - timedelta(days=365 * 2)\n",
+    "end_date = today - timedelta(days=1)\n",
+    "\n",
+    "# Create the request object\n",
+    "req = CryptoBarsRequest(\n",
+    "    symbol_or_symbols = [symbol],\n",
+    "    timeframe=TimeFrame.Day,\n",
+    "    start=start_date,\n",
+    "    end_date=end_date,\n",
+    ")\n",
+    "\n",
+    "bars_date = crypto_historical_data_client.get_crypto_bars(req).df.reset_index(level='symbol', drop=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df = bars_date.copy()\n",
+    "df.head()"
```

Comment:
How about adding a comment to explain the reason of using `.copy()` here?
In my understanding, if we copy here, we do not need to fetch data (i.e. bars_date) again within same session. It might make easy to change and check below codes if user want to tweak.